docs/building.md

```diff
@@ -8,12 +8,12 @@ video drivers for your card.
 
 ### Windows
 
-* Windows 8 or 8.1
+* Windows 8 or later
 * Visual Studio 2015 or Visual Studio 2017
-* [Python 2.7](https://www.python.org/downloads/release/python-2713/)
+* [Python 3.6](https://www.python.org/downloads/release/python-362/)
```

Comment:
@drillsar PRs should be made with only the main repo in mind, not assuming other PRs have been pulled as well.